Tests/WorkspaceTests/PinsStoreTests.swift

```diff
@@ -31,7 +31,19 @@ final class PinsStoreTests: XCTestCase {
         let pin = PinsStore.Pin(package: foo, repository: fooRepo, state: state, reason: "bad")
         // We should be able to round trip from JSON.
         XCTAssertEqual(try PinsStore.Pin(json: pin.toJSON()), pin)
-        
+        mktmpdir { tmpPath in
```

Comment:
As I said, since we don't need to do filesystem operations, we don't need to create directories. Passing dummy paths will work.